<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HMSExtractorDialogBase</class>
 <widget class="QDialog" name="HMSExtractorDialogBase">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>520</width><height>280</height></rect>
  </property>
  <property name="windowTitle"><string>HMS Extractor</string></property>

  <layout class="QVBoxLayout" name="verticalLayout">

   <!-- GRID PRINCIPAL -->
   <item>
    <layout class="QGridLayout" name="gridLayout">

     <!-- Carpeta HMS -->
     <item row="0" column="0">
      <widget class="QLabel" name="labelFolder">
       <property name="text"><string>Carpeta del proyecto HMS:</string></property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditFolder"/>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="btnBrowse">
       <property name="text"><string>...</string></property>
      </widget>
     </item>

     <!-- BASIN -->
     <item row="1" column="0">
      <widget class="QLabel" name="labelBasin">
       <property name="text"><string>Archivo .basin:</string></property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="comboBasin"/>
     </item>

     <!-- Carpeta salida -->
     <item row="2" column="0">
      <widget class="QLabel" name="labelOutput">
       <property name="text">
        <string>Carpeta de salida (se crea dentro de la carpeta HMS):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditOut"/>
     </item>

     <!-- OPCIONES -->
     <item row="3" column="0" colspan="3">
      <widget class="QGroupBox" name="groupOptions">
       <property name="title"><string>Opciones</string></property>
       <layout class="QVBoxLayout" name="verticalLayoutOptions">

        <item>
         <widget class="QCheckBox" name="checkLoadElements">
          <property name="text">
           <string>Cargar elementos HMS</string>
          </property>
          <property name="checked"><bool>true</bool></property>
         </widget>
        </item>

        <item>
         <widget class="QCheckBox" name="checkLoadBackground">
          <property name="text">
           <string>Cargar shapefiles del Background Layer</string>
          </property>
          <property name="checked"><bool>true</bool></property>
         </widget>
        </item>

       </layout>
      </widget>
     </item>

    </layout>
   </item>

   <!-- BOTONES -->
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item><spacer name="spacer"/></item>
     <item><widget class="QPushButton" name="btnRun"><property name="text"><string>Extraer</string></property></widget></item>
     <item><widget class="QPushButton" name="btnClose"><property name="text"><string>Cancelar</string></property></widget></item>
    </layout>
   </item>

  </layout>
 </widget>

 <resources/>
 <connections/>
</ui>
